tests/datatypes/test_str.py

```diff
@@ -30,13 +30,27 @@ def test_isspace(self):
                 print(s.isspace())
             """)
 
+    def test_isalnum(self):
+        self.assertCodeExecution("""
+            for word in ["", "12", "abc", "abc12", "\u00c4", "\x41", "a@g", "äÆ",
```

Comment:
Since `alnum()` seems to be effectively some combination of `isalpha()` and `isdecimal()`, probably should include some strings with uppercase letters here, to ensure that uppercase letters don't make strings non-`alnum` when they should be.